guides/user/managed-delivery/artifacts/index.md

```diff
@@ -2,7 +2,8 @@
 layout: single
 title:  "Artifacts"
 sidebar:
-  nav: reference
+  nav: guides
+redirect_from: /reference/managed-delivery/artifacts/
```

Comment:
Do we need the redirects? The content was up on spinnaker.io/reference/ only for a little while, right?

Reply:
Well, some of the pages were up for longer and are linked from the spinnaker UI (like the resource status page). I figured, why not put redirects for all of them, we have that functionality.

Reply:
:+1: